src/main/run_work.py

```diff
@@ -19,10 +19,18 @@ def _check_config(path):
 
 
 def _check_draft(path,draft_config):
-    if SaveApiKey().exsitapikey() and not os.path.exists(os.path.join(path, "ta", "draft.json")) and not draft_config:
-        return False
-    elif not SaveApiKey().exsitapikey() and not os.path.exists(os.path.join(path, "ta", "draft.json")):
-        return False
+    # Not have draft.json
+    if not os.path.exists(os.path.join(path, "ta", "draft.json")):
+        # Have an api-key
+        if SaveApiKey().exsitapikey():
```

Comment:
1. Fix typo. Maybe changing the name of this method should be done also to make it easier to understand.
2. Why are you creating the new instance of SaveApiKey but just use only one method here.
> This can means that this method is not depend on the data in the SaveApiKey object. So consider changing it to be an `@staticmethod` so you can call this method without creating the new instance.

Reply:
@Tanidean Can I change the SaveApiKey method to the static method.